2019/src/components/header.js

```diff
@@ -1,34 +1,46 @@
 import { Link } from "gatsby"
 import PropTypes from "prop-types"
 import React from "react"
+import LogoText from "../images/logo-text"
+import { Box, Title } from "./layout-components"
+import styled from "@emotion/styled"
 
-const Header = ({ siteTitle }) => (
-  <header
-    style={{
-      background: `rebeccapurple`,
-      marginBottom: `1.45rem`,
-    }}
-  >
+const StyledHeader = Box.withComponent("header")
+
+const StyledLink = styled(Link)`
+  display: block;
+  width: 200px;
+  text-decoration: none;
+  background-image: linear-gradient(transparent 92%, #2160ad 0);
+`
+
+const Header = () => (
+  <StyledHeader marginBottom="4">
     <div
       style={{
         margin: `0 auto`,
         maxWidth: 960,
         padding: `1.45rem 1.0875rem`,
       }}
     >
-      <h1 style={{ margin: 0 }}>
-        <Link
-          to="/"
-          style={{
-            color: `white`,
-            textDecoration: `none`,
-          }}
-        >
-          {siteTitle}
-        </Link>
-      </h1>
+      <Title
```

Comment:
might be good to make a VisuallyHidden component - I'm sure we'll use this again. we can do that later though

Reply:
thanks yeah, good idea, i'll do that later.